### PR TITLE
Integrate image slider paragraph. See #117.

### PR DESCRIPTION
**Pull request by corneaalina**

### CODE REVIEW

File: docroot/themes/custom/dct_theme/dct_theme.theme [hunk 1]
```diff
@@ -13,3 +13,14 @@ function dct_theme_preprocess_page(&$variables) {
     $variables['#attached']['library'][] = 'dct_theme/parallax';
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK() for image slider paragraph.
```
Comment (lauracernau) [inline docroot/themes/custom/dct_theme/dct_theme.theme]: Preprocess for paragraph templates, you can add actions for other types of paragraphs.

File: docroot/themes/custom/dct_theme/js/image-slider.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/**
+ * @file
+ * DCT image slider behaviors.
+ */
+
+(function($, Drupal) {
+
+    'use strict';
+
+
```
Comment (lauracernau) [inline docroot/themes/custom/dct_theme/js/image-slider.js]: No need for this extra line here.

File: docroot/themes/custom/dct_theme/dct_theme.libraries.yml [hunk 3]
```diff
@@ -30,3 +30,27 @@ parallax:
     js/parallax.js: {}
   dependencies:
     - core/jquery
+
+slick:
+  version: 1.x
+  js:
+    js/lib/slick.min.js: {}
```
Comment (danielpopdan) [inline docroot/themes/custom/dct_theme/dct_theme.libraries.yml]: I think this one is managed by node.

File: docroot/themes/custom/dct_theme/templates/paragraphs/paragraph--image-slider-paragraph--default.html.twig [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<div class="location-slider">
+    <div class="l-container">
+        <div id="location-slider">
+            {%  for image in content.field_image['#items'] %}
+                <div class="location-slider--slide">
+                    <img src="{{ file_url(image.entity.uri.value) }}" alt="{{ image.alt }}" data-action="zoom">
```
Comment (danielpopdan) [inline docroot/themes/custom/dct_theme/templates/paragraphs/paragraph--image-slider-paragraph--default.html.twig]: This is not ok. The image should be rendered entirely. Please add data-action in a preprocess.

File: docroot/themes/custom/dct_theme/dct_theme.libraries.yml (hunk 3 above)
Comment (AlexandruSateanuHoria) [reply docroot/themes/custom/dct_theme/dct_theme.libraries.yml]: After you run npm install command in the dct_theme folder, you can set the path for slick here to '/node_modules/slick-carousel/slick/slick.min.js'

**Pull request merged**